src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -95,9 +91,12 @@ public void onTestFailure(ITestResult result) {
     public void onTestSkipped(ITestResult result) {
         logger.info(result.getName() + " skipped...");
         try {
-            appendLogToAllure();
-            clearFile();
-        } catch (IOException e) {
+            allureUtil.appendLogToAllure();
+            allureUtil.clearFile();
+            allureUtil.attachScreenshot();
+            allureUtil.attachVideoMP4(result);
+            allureUtil.deleteFolderWithVideo(result);
+        } catch (IOException | NullPointerException e) {
```

Comment:
why do you catch NullPointerException? it's a really bad idea, it might cause lots of problems, you need to try to avoid NLPE

Reply:
Ooops :) Deleted